### PR TITLE
Add vendored `plist` to ReadMe.

### PR DESCRIPTION
**Pull request by reitermarkus**

### CODE REVIEW

File: Library/Homebrew/vendor/README.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Vendored Dependencies
 =====================
 
-* [okjson](https://github.com/kr/okjson), version 43.
+* [okjson](https://github.com/kr/okjson), version 43
+
+* [plist](https://github.com/bleything/plist), version unknown
```
Comment (MikeMcQuaid) [inline Library/Homebrew/vendor/README.md]: Glad to have this but the `version unknown` worries me a bit. Could we maybe try to bump it to a known revision at some point?

**Comment by MikeMcQuaid** [comment]
Thanks! 🚢

File: Library/Homebrew/vendor/README.md (hunk 1 above)
Comment (reitermarkus) [inline Library/Homebrew/vendor/README.md]: Will look into it. Let's leave this open for now so I don't forget.

Comment (MikeMcQuaid) [inline Library/Homebrew/vendor/README.md]: 🆒 thanks.

**Comment by UniqMartin** [comment]
I guess that's just an oversight here, but try to create the branches for PRs in your fork instead of the repositories in the Homebrew organization. One way to enforce this in the future is to set the push URL of the `origin` remote (`git remote set-url --push origin <url>`) to an invalid URL or the shared fetch/push URL to a read-only location.

**Comment by reitermarkus** [comment]
@UniqMartin, yes that was an oversight, I started this on my iPad with the ”edit“ button on GitHub.

File: Library/Homebrew/vendor/README.md [hunk 2]
```diff
@@ -1,7 +1,9 @@
 Vendored Dependencies
 =====================
 
-* [okjson](https://github.com/kr/okjson), version 43.
+* [okjson](https://github.com/kr/okjson), version 43
+
+* [plist](https://github.com/bleything/plist), version 3.1.0
```
Comment (MikeMcQuaid) [inline Library/Homebrew/vendor/README.md]: 😍

**Comment by MikeMcQuaid** [comment]
Nice work 👍 🚢

**Comment by reitermarkus** [comment]
Merging this, as only some unrelated tests failed on travis. Also making the push URL read-only now. 😄

**Pull request merged**